Auth0/SafariWebAuth.swift

```diff
@@ -120,7 +120,11 @@ class SafariWebAuth: WebAuth {
         let authorizeURL = self.buildAuthorizeURL(withRedirectURL: redirectURL, defaults: handler.defaults, state: state)
 
         #if swift(>=3.2)
-        if #available(iOS 11.0, *), self.authenticationSession {
+        if #available(iOS 12.0, *), self.authenticationSession {
+            let session = ASAuthenticationSession(authorizeURL: authorizeURL, redirectURL: redirectURL, state: state, handler: handler, finish: callback, logger: logger)
+            logger?.trace(url: authorizeURL, source: "AuthenticationServicesAuthenticationSession")
+            self.storage.store(session)
+        } else if #available(iOS 11.0, *), self.authenticationSession {
```

Comment:
For iOS 11 you still have to use SFAuthenticationSession as per Apple guidelines. So there are 3 tiers of support iOS >=12, iOS 11, iOS <11.0

Reply:
My mistake, saw the word `Safari`, naming these classes/methods is quite tricky 😆